www/c-icap/src/opnsense/mvc/app/models/OPNsense/CICAP/Menu/Menu.xml

```diff
@@ -0,0 +1,5 @@
+<menu>
+  <Services>
+    <C-ICAP cssClass="fa fa-ambulance" url="/ui/cicap/general/index" />
```

Comment:
this may make PHP cringe in the future, better set "<C_ICAP VisibleName="C-ICAP" ...>"